### PR TITLE
Correct sort order and adds missing No-IP domains to the PSL.

### PR DESCRIPTION
**Pull request by noipcom**

### CODE REVIEW

**Comment by noipcom** [comment]
At No-IP we allow anyone to create a hostname attached to our domain names. We believe since anyone can create a name attached to our domains our domains should be listed on Public Suffix List. This would add to the public domains already on the list. We also plan on keeping the domains public. Majority of our domains have been around since 2001 and we will continue to offer hostname creation on our domains listed. 

We’ve recently added 6 new domains which is one of the reasons for this PR. 

Note: Our previous PR was already merged and included 85 of our domains. After review of the guidelines I realized we did not sort correctly previously. The 6 new domains are access.ly, freedynamicdns.net, freedynamicdns.org, hosthampster.com, myddns.me, viewdns.net and has the TXT records added. The previous domains still have the previous pull request, 64, let me know if this is an issue.

```
_psl.access.ly
"https://github.com/publicsuffix/list/pull/682"
_psl.freedynamicdns.net
"https://github.com/publicsuffix/list/pull/682"
_psl.freedynamicdns.org
"https://github.com/publicsuffix/list/pull/682"
_psl.hosthampster.com
"https://github.com/publicsuffix/list/pull/682"
_psl.myddns.me
"https://github.com/publicsuffix/list/pull/682"
_psl.viewdns.net
"https://github.com/publicsuffix/list/pull/682"
```

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12081,91 +12081,97 @@ nerdpol.ovh
...
 servegame.com
 servehalflife.com
 servehttp.com
 serveirc.com
-serveminecraft.net
 servemp3.com
 servepics.com
 servequake.com
-sytes.net
+hosthampster.com
```
Comment (sleevi) [inline public_suffix_list.dat]: Wrong sort.

File: public_suffix_list.dat [hunk 2]
```diff
@@ -12081,91 +12081,97 @@ nerdpol.ovh
...
+no-ip.info
+access.ly
+brasilia.me
+ddns.me
+dnsfor.me
+hopto.me
+loginto.me
+noip.me
 webhop.me
+myddns.me
```
Comment (sleevi) [inline public_suffix_list.dat]: Wrong sort

File: public_suffix_list.dat [hunk 3]
```diff
@@ -12081,91 +12081,97 @@ nerdpol.ovh
...
+mydissent.net
+myeffect.net
+mymediapc.net
+mypsx.net
+mysecuritycamera.net
+nhlfan.net
+no-ip.net
+pgafan.net
+privatizehealthinsurance.net
+bounceme.net
```
Comment (sleevi) [inline public_suffix_list.dat]: wrong sorts

File: public_suffix_list.dat [hunk 4]
```diff
@@ -12081,91 +12081,97 @@ nerdpol.ovh
...
+viewdns.net
+cable-modem.org
+collegefan.org
+couchpotatofries.org
+mlbfan.org
+mysecuritycamera.org
+nflfan.org
+read-books.org
+ufcfan.org
+hopto.org
```
Comment (sleevi) [inline public_suffix_list.dat]: Wrong sorts

**Comment by sleevi** [comment]
Please review your change. I only see three domains referring to PR 682 - so it's most likely you've inadvertently deleted some.

**Comment by noipcom** [comment]
@sleevi Sorry about that. Corrected the sorting. Verified all the TXT records. Confirmed there are 6 new domains being added with this pull request. 

```
_psl.mmafan.biz
"https://github.com/publicsuffix/list/pull/64"
_psl.myftp.biz
"https://github.com/publicsuffix/list/pull/64"
_psl.no-ip.biz
"https://github.com/publicsuffix/list/pull/64"
_psl.no-ip.ca
"https://github.com/publicsuffix/list/pull/64"
_psl.fantasyleague.cc
"https://github.com/publicsuffix/list/pull/64"
_psl.gotdns.ch
"https://github.com/publicsuffix/list/pull/64"
_psl.3utilities.com
"https://github.com/publicsuffix/list/pull/64"
_psl.blogsyte.com
"https://github.com/publicsuffix/list/pull/64"
_psl.ciscofreak.com
"https://github.com/publicsuffix/list/pull/64"
_psl.damnserver.com
"https://github.com/publicsuffix/list/pull/64"
_psl.ddnsking.com
"https://github.com/publicsuffix/list/pull/64"
_psl.ditchyourip.com
"https://github.com/publicsuffix/list/pull/64"
_psl.dnsiskinky.com
"https://github.com/publicsuffix/list/pull/64"
_psl.dynns.com
"https://github.com/publicsuffix/list/pull/64"
_psl.geekgalaxy.com
"https://github.com/publicsuffix/list/pull/64"
_psl.health-carereform.com
"https://github.com/publicsuffix/list/pull/64"
_psl.homesecuritymac.com
"https://github.com/publicsuffix/list/pull/64"
_psl.homesecuritypc.com
"https://github.com/publicsuffix/list/pull/64"
_psl.hosthampster.com
"https://github.com/publicsuffix/list/pull/682"
_psl.myactivedirectory.com
"https://github.com/publicsuffix/list/pull/64"
_psl.mysecuritycamera.com
"https://github.com/publicsuffix/list/pull/64"
_psl.myvnc.com
"https://github.com/publicsuffix/list/pull/64"
_psl.net-freaks.com
"https://github.com/publicsuffix/list/pull/64"
_psl.onthewifi.com
"https://github.com/publicsuffix/list/pull/64"
_psl.point2this.com
"https://github.com/publicsuffix/list/pull/64"
_psl.quicksytes.com
"https://github.com/publicsuffix/list/pull/64"
_psl.securitytactics.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servebeer.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servecounterstrike.com
"https://github.com/publicsuffix/list/pull/64"
_psl.serveexchange.com
"https://github.com/publicsuffix/list/pull/64"
_psl.serveftp.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servegame.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servehalflife.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servehttp.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servehumour.com
"https://github.com/publicsuffix/list/pull/64"
_psl.serveirc.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servemp3.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servep2p.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servepics.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servequake.com
"https://github.com/publicsuffix/list/pull/64"
_psl.servesarcasm.com
"https://github.com/publicsuffix/list/pull/64"
_psl.stufftoread.com
"https://github.com/publicsuffix/list/pull/64"
_psl.unusualperson.com
"https://github.com/publicsuffix/list/pull/64"
_psl.workisboring.com
"https://github.com/publicsuffix/list/pull/64"
_psl.dvrcam.info
"https://github.com/publicsuffix/list/pull/64"
_psl.ilovecollege.info
"https://github.com/publicsuffix/list/pull/64"
_psl.no-ip.info
"https://github.com/publicsuffix/list/pull/64"
_psl.access.ly
"https://github.com/publicsuffix/list/pull/682"
_psl.brasilia.me
"https://github.com/publicsuffix/list/pull/64"
_psl.ddns.me
"https://github.com/publicsuffix/list/pull/64"
_psl.dnsfor.me
"https://github.com/publicsuffix/list/pull/64"
_psl.hopto.me
"https://github.com/publicsuffix/list/pull/64"
_psl.loginto.me
"https://github.com/publicsuffix/list/pull/64"
_psl.myddns.me
"https://github.com/publicsuffix/list/pull/682"
_psl.noip.me
"https://github.com/publicsuffix/list/pull/64"
_psl.webhop.me
"https://github.com/publicsuffix/list/pull/64"
_psl.bounceme.net
"https://github.com/publicsuffix/list/pull/64"
_psl.ddns.net
"https://github.com/publicsuffix/list/pull/64"
_psl.eating-organic.net
"https://github.com/publicsuffix/list/pull/64"
_psl.freedynamicdns.net
"https://github.com/publicsuffix/list/pull/682"
_psl.mydissent.net
"https://github.com/publicsuffix/list/pull/64"
_psl.myeffect.net
"https://github.com/publicsuffix/list/pull/64"
_psl.mymediapc.net
"https://github.com/publicsuffix/list/pull/64"
_psl.mypsx.net
"https://github.com/publicsuffix/list/pull/64"
_psl.mysecuritycamera.net
"https://github.com/publicsuffix/list/pull/64"
_psl.nhlfan.net
"https://github.com/publicsuffix/list/pull/64"
_psl.no-ip.net
"https://github.com/publicsuffix/list/pull/64"
_psl.pgafan.net
"https://github.com/publicsuffix/list/pull/64"
_psl.privatizehealthinsurance.net
"https://github.com/publicsuffix/list/pull/64"
_psl.redirectme.net
"https://github.com/publicsuffix/list/pull/64"
_psl.serveblog.net
"https://github.com/publicsuffix/list/pull/64"
_psl.serveminecraft.net
"https://github.com/publicsuffix/list/pull/64"
_psl.sytes.net
"https://github.com/publicsuffix/list/pull/64"
_psl.viewdns.net
"https://github.com/publicsuffix/list/pull/682"
_psl.cable-modem.org
"https://github.com/publicsuffix/list/pull/64"
_psl.collegefan.org
"https://github.com/publicsuffix/list/pull/64"
_psl.couchpotatofries.org
"https://github.com/publicsuffix/list/pull/64"
_psl.freedynamicdns.org
"https://github.com/publicsuffix/list/pull/682"
_psl.hopto.org
"https://github.com/publicsuffix/list/pull/64"
_psl.mlbfan.org
"https://github.com/publicsuffix/list/pull/64"
_psl.myftp.org
"https://github.com/publicsuffix/list/pull/64"
_psl.mysecuritycamera.org
"https://github.com/publicsuffix/list/pull/64"
_psl.nflfan.org
"https://github.com/publicsuffix/list/pull/64"
_psl.no-ip.org
"https://github.com/publicsuffix/list/pull/64"
_psl.read-books.org
"https://github.com/publicsuffix/list/pull/64"
_psl.ufcfan.org
"https://github.com/publicsuffix/list/pull/64"
_psl.zapto.org
"https://github.com/publicsuffix/list/pull/64"
_psl.no-ip.co.uk
"https://github.com/publicsuffix/list/pull/64"
_psl.golffan.us
"https://github.com/publicsuffix/list/pull/64"
_psl.noip.us
"https://github.com/publicsuffix/list/pull/64"
_psl.pointto.us
"https://github.com/publicsuffix/list/pull/64"
```

**Comment by sleevi** [comment]
_psl.myddns.me has no TXT record
_psl.viewdns.net has no TXT record
_psl.couchpotatofries.org has no TXT record
_psl.freedynamicdns.org has no TXT record
_psl.zapto.org has no TXT record

**Pull request closed without merging**